source/userguide/entity_resolver.rst

```diff
@@ -210,8 +210,40 @@ Upon building all the required resolvers, to access a particular resolver, adapt
 
   <ElasticsearchEntityResolver ready: True, dirty: False>
 
-Access & train an entity resolver as standalone component
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+(Optional) Access & train specific entity resolvers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As an alternative to building all entity resolvers required for an app, you can also access and train resolvers of a particular domain-intent hierarchy. This might be useful in case if you wish to run some simple tests for specific resolvers.
```

Comment:
```suggestion
As an alternative to building all entity resolvers required for an app, you can also access and train resolvers of a particular domain-intent hierarchy. This might be useful in case you want to to run some simple tests for specific resolvers.
```